CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 * [#3394](https://github.com/bbatsov/rubocop/issues/3394): Remove `Style/TrailingCommmaInLiteral` in favor of two new cops. ([@garettarrowood][])
 
 ## 0.52.1 (2017-12-27)
+* [#5319](https://github.com/bbatsov/rubocop/pull/5319): Add new `Security/Open` cop. ([@mame][])
```

Comment:
This should be under `New Feature` higher up in the file.

Reply:
Sorry!  I left it to git merge. Fixed.